nni/algorithms/compression/v2/pytorch/base/compressor.py

```diff
@@ -3,7 +3,7 @@
 
 import collections
 import logging
-from typing import List, Dict, Optional, Tuple, Any
+from typing import Any, List, Dict, OrderedDict, Optional, Tuple
```

Comment:
How is `OrderedDict` get used? In Python 3.7+ ordinary `dict` has guaranteed order. I think `OrderedDict` is not very useful in most cases now.

Reply:
just want a guaranteed order, I will change it to `dict`.